src/reactpy_django/utils.py

```diff
@@ -291,15 +294,15 @@ def django_query_postprocessor(
     return data
 
 
-def func_has_params(func: Callable) -> bool:
-    """Checks if a function has any args or kwarg parameters."""
+def func_has_args(func: Callable) -> bool:
+    """Checks if a function has any args or kwarg."""
     signature = inspect.signature(func)
 
     # Check if the function has any args/kwargs
-    return str(signature) != "()" or True
+    return str(signature) != "()"
```

Comment:
If this isn't used in multiple places perhaps we can just embed the logic where it's currently being used.

Reply:
This comment actually made me realize I forgot to add in a check for this in the template tag.

But yes, it should be used in at minimum two locations. The consumer and the template tag.

I will add a regression test.